### PR TITLE
Pop up window

### PR DESCRIPTION
**Pull request by lawrahkonwea**
In this project, the popup window was created using ;
- Html
-  CSS
-  Javascript.
-   Javascript object was used to store the data
-   Popup window was implemented for both desktop and mobile screen sizes
-   When a user taps on the button to check project details, the popup with details about the project appears.
-  When the user clicks on the close button the popup disappears

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,47 +10,47 @@
...
+            <li><a href="https://www.linkedin.com/in/laura-konwea-99647a23a/"><img src="images/Linkedin icon.jpg"
+                  alt="linkedin"></a></li>
+            <li><a href="#"><img src="images/medium.jpg" alt="medium"></a></li>
+            <li><a href="https://github.com/lawrahkonwea"><img src="images/github.jpg" alt="github"></a></li>
+            <li><a href="#"><img src="images/angellist.jpg" alt="angellist"></a></li>
+          </ul>
+        </div>
+      </div>
+    </section>
     <section class="project-main-wrapper1">
```
Comment (Olamarx) [inline index.html:54]: - [ ] Kindly display the whole of this project section dynamically with javascript. By doing this, you will only need to add the data in the array in the future when you want to add a new project. So to do this, you will have to create an array, just as you did in the pop-up. Then iterate on it with a loop. Kindly remember all the files in the HTML file too.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️ 
- Pop-up functionalities ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html:54]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @lawrahkonwea 

Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**